test/e2e/network/network_policy.go

```diff
@@ -312,6 +314,52 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			})
 		})
 
+		ginkgo.It("should allow ingress access from namespace-b on one named port [Feature:NetworkPolicy]", func() {
```

Comment:
The test name should be more generic - EG `should allow ingress access from namespace on one named port`.